keps/sig-node/4540-strict-cpu-reservation/kep.yaml

```diff
@@ -32,7 +31,7 @@ milestone:
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: "CPUManagerPolicyAlphaOptions"                                                
+  - name: "CPUManagerPolicyAlphaOptions"
```

Comment:
Shouldn't we be adding CPUManagerPolicyBetaOptions here?

Reply:
I was not sure. I checked the other similar CPUManager KEPs in beta stage, they don't have the BetaOptions under feature-gates plus the comment saying for alpha, so I followed the flow. I guess there is nothing wrong to add it, I will do a change to add it.

Reply:
The first version is one feature gate "CPUManagerPolicyBetaOptions" because one policy option is under one feature gate in CPUManager. I will change back to the first version.